xclim/ensembles/_robustness.py

```diff
@@ -247,7 +293,7 @@ def change_significance(
             **das,
         ),
     )
-    return change_frac, pos_frac
+    return changed, change_frac, pos_frac
```

Comment:
```suggestion
    return change_frac, pos_frac, changed
```
This is a breaking change, but I think the impact is lower if the order is unchanged. 
Also could you add something to the docstring about this new return value?